base_attachment_object_storage/README.rst

```diff
@@ -38,3 +38,10 @@ Default configuration means:
   stored in database
 * application/javascript are stored in database whatever their size
 * text/css are stored in database whatever their size
+
+Unsafe mode
+-----------
+
+Define a environment variable `ATTACHMENT_STORAGE_UNSAFE` set to `1`
+This will prevent and avoid any king of exceptions and filestore read/write
```

Comment:
```suggestion
This will prevent any kind of exceptions and any read/write on filestore 
```

Reply:
Hey, if I read "unsafe", I read it as it "activate the unsafe mode in which they ar not guarantees", which means kind of the reverse of what it is doing.
Maybe it's only me.

Reply:
> Hey, if I read "unsafe", I read it as it "activate the unsafe mode in which they ar not guarantees", which means kind of the reverse of what it is doing. Maybe it's only me.

Hi @guewen
Meaning of unsafe could be a lot of interpretations in fact, but you're right. In this case "unsafe" means that you can't garrantee filestore interactions as it is almost totally deactivated. Which means you can't read or write on a storage that can't be accessible, which is not safe to use in this context because of possible data loss.

Have you a propasal for this definition then ?  INACTIVE or something ?

Reply:
And it was made to follow the same logic as the unsafe mode of the cloud_platform module

Reply:
Changed for "inactive" concept instead of "unsafe" !

Reply:
@StephaneMangin thanks, checked on customers project

Reply:
The unsafe param in the cloud platform module does not disable any functionality, it allows to start odoo even if some checks would not be valid, that's not really the same idea then.

Disable is better 👍

Reply:
I like `DISABLE`, it makes the thing clear about the intent.

Reply:
Done !